packages/components/psammead-radio-schedule/src/StartTime/index.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { number, string, shape, oneOf } from 'prop-types';
+import { GEL_SPACING_HLF, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { C_RHINO, C_PEBBLE } from '@bbc/psammead-styles/colours';
+import { getMinion } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { coreIcons } from '@bbc/psammead-assets/svgs';
+import TimestampContainer from '@bbc/psammead-timestamp-container';
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledClock = styled.span`
+  display: flex;
+  align-items: center;
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-right: ${GEL_SPACING_HLF}; padding-left: ${GEL_SPACING};`
+      : `padding-left: ${GEL_SPACING_HLF}; padding-right: ${GEL_SPACING};`}
+  > svg {
+    color: ${C_RHINO};
+    margin: 0;
+  }
+`;
+
+const ClockIcon = ({ dir }) => {
+  return <StyledClock dir={dir}>{coreIcons.clock}</StyledClock>;
+};
+
+ClockIcon.propTypes = {
+  dir: oneOf(['ltr', 'rtl']),
+};
+
+ClockIcon.defaultProps = {
+  dir: 'ltr',
+};
+
+const StyledTimestamp = styled.span`
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+
+  > time {
+    color: ${C_RHINO};
+    ${({ script }) => script && getMinion(script)}
+    ${({ service }) => service && getSansRegular(service)}
+  }
+
+  &::after {
+    content: '';
+    border-top: 0.0625rem solid ${C_PEBBLE};
+    top: ${({ script }) => 0.5 + script.minion.groupA.lineHeight / 2 / 16}rem;
+    ${({ dir }) =>
+      dir === 'ltr' ? `margin-left: 0.625rem;` : `margin-right: 0.625rem;`}
+    width: 100vw;
```

Comment:
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/43134742/73365961-0fc7a280-42a5-11ea-8ac5-24427e3fc020.png">
This is causing the screen to be horizontally scrollable (note the scrollbar on the bottom) - the `:after` pseudo-element is the full width of the screen, then the clock and time are also added on top of that full width. This should be possible to remedy with flex but I don't know the css off the top of my head.

Reply:
From a quick investigation, putting `width: 100%` on the `StyledTimestamp` and changing the `width: 100vw` to `width: 100%` on the `:after` should solve this.

Reply:
Good spot, fixed now 👍